src/main.js

```diff
@@ -61,9 +61,9 @@ const TRACK_CHANGES_HOURS_INTERVAL = 24;
   }
 
   logger.info('The scheduler is running…');
+  logger.info('Documents will be tracked at minute 30 past hour 03, 09, 15, and 21 every day');
 
-  logger.info(`Documents will be tracked every ${TRACK_CHANGES_HOURS_INTERVAL} hours\n`);
-  setInterval(async () => {
+  cron('30 3,9,15,21 * * *', async () => {
     await archivist.trackChanges(serviceIds);
```

Comment:
Why would we `await` this? 🤔 
```suggestion
    archivist.trackChanges(serviceIds);
```